lib/evss/auth_headers.rb

```diff
@@ -22,8 +22,9 @@ def to_h
         'va_eauth_pid' => @user.participant_id,
         'va_eauth_pnid' => @user.ssn,
         'va_eauth_birthdate' => iso8601_birth_date,
-        'va_eauth_authorization' => eauth_json
-      )
+        'va_eauth_authorization' => eauth_json,
+        'va_eauth_authenticationauthority' => 'eauth'
+      ).delete_if { |_k, v| v.blank? }
```

Comment:
I was the one that recommended this approach but want to make certain that we don't break anything from the va.gov side of the house when using this.

Reply:
I'm not sure whether removing blank headers will disrupt va.gov's use of EVSS. Assuming that is OK and we want to continue removing blank headers though, perhaps `sanitize` could be rewritten to remove keys with either nil or empty string values. As it works now `sanitize` will convert nil to an empty string and then we'll remove the keys will empty strings here.

Reply:
There's a minimum set of headers required for auth, of the non-hardcoded headers they are:
```
va_eauth_firstName
va_eauth_lastName
va_eauth_birthdate
va_eauth_pid
va_eauth_pnid
va_eauth_dodedipnid
```
FWIW EVSS have previously confirmed that, unless required, blank headers are ignored, although for us that's usually only `va_eauth_birlsfilenumber`. Agree with @dvogel that if we're going to remove the blank ones that we roll it up into the sanitize method.

Reply:
We are running into issue with EVSS causing 500 errors when we send empty pid's. Our contact at EVSS said the empty values what was the issue. Have you all seen anything like that before? It could be a little bit of a non issue now that we are pulling missing data from MVI.

Reply:
I've never run into that specific issue, we did have an issue where `va_eauth_dodedipnid` was not included and we got 401s.

Reply:
I ended up removing this delete based on conversations with @bastosmichael.  Now that we are pulling in data from MVI.